src/services/service.js

```diff
@@ -264,6 +266,22 @@ class Service {
     })
 
   }
+
+  execGraphQLQuery(projectId, graphqlQuery) {
+    return new Promise((resolve, reject) => {
+      const client = new ApolloClient({
+        uri: `/v1/api/graphql/${projectId}`,
+      });
+      client
```

Comment:
Send graphql variables as well here which we get from grapqhlParams as shown here - https://stackoverflow.com/questions/51522902/apollo-query-with-variable